pkg/backend/backend.go

```diff
@@ -316,9 +316,10 @@ type QueryOperation struct {
 
 // StackConfiguration holds the configuration for a stack and it's associated decrypter.
 type StackConfiguration struct {
-	Environment esc.Value
-	Config      config.Map
-	Decrypter   config.Decrypter
+	EnvironmentImports []string
+	Environment        esc.Value
+	Config             config.Map
+	Decrypter          config.Decrypter
```

Comment:
I think at this point it'd probably be beneficial to add some doc comments here -- for people new to the codebase or unfamiliar with Pulumi, I think it'd otherwise be natural to assume that `Environment*` was talking about e.g. the OS environment.